README.md

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/wrapper-validation-action@releases/v1
```

Comment:
:x: why change this to point to a branch ?

Reply:
That was unintentional and not part of the change I meant to make.